stack/.env

```diff
@@ -0,0 +1,10 @@
+SSO_BASE_URL=http://keycloak:8080/auth
```

Comment:
Did you think about alternatives for `keycloak`?
Think it might be a problem as with this every developer need at least one time local administration rights.

Reply:
>  local administration rights

why?

Reply:
`keycloak` needs to be added to hosts file that the login redirect works.

Reply:
yeah okay, i understand what you mean. I will look into it. Changing the hosts file just to make it work is rather bad